### PR TITLE
Spanish

### PR DESCRIPTION
**Pull request by faustinoloeza**
My translation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2163,6 +2163,7 @@ This is also available in other languages:
 
   - ![br](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/Brazil.png) **Brazilian Portuguese**: [fesnt/clean-code-javascript](https://github.com/fesnt/clean-code-javascript)
   - ![es](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/Uruguay.png) **Spanish**: [andersontr15/clean-code-javascript](https://github.com/andersontr15/clean-code-javascript-es)
+  - ![cn](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/Mexico.png) **Spanish**: [faustinoloeza/clean-code-javascript](https://github.com/ryanmcdermott/clean-code-javascript)
```
Comment (ryanmcdermott) [inline README.md]: This should say `![es]`. The `cn` is for China I believe

Comment (ryanmcdermott) [inline README.md]: This is wrong, it is linking back to my repo. Link this to yours: https://github.com/faustinoloeza/clean-code-javascript

And please change `cn` to `es`

Comment (faustinoloeza) [inline README.md]: I sent another PR

**Comment by felquis** [comment]
@ryanmcdermott you can close this PR in favor of #224
